subprojects/core/src/main/java/org/gradle/process/internal/JavaExecHandleBuilder.java

```diff
@@ -58,9 +58,11 @@
 
 /**
  * Use {@link JavaExecHandleFactory} instead.
+ *
+ * TODO: Make this class Provider API agnostic in the same way as {@link DefaultClientExecHandleBuilder} is.
```

Comment:
❓ When do we plan to do this? Can we add an issue for it?

Reply:
I think we can also remove this TODO. There is no need to do that other then reflecting `Provider API agnostic` level of `DefaultClientExecHandleBuilder`.

Reply:
Removed